### PR TITLE
fix(docker): fix docker-compose file was not working

### PR DESCRIPTION
**Pull request by baptistemesta**

### CODE REVIEW

**Comment by baptistemesta** [comment]
`2022.1` version need to change because it's not working with the removal of db-tool

File: modules/runtime/pages/bonita-docker-installation.adoc [hunk 1]
```diff
@@ -168,25 +168,23 @@ services:
       - |
         set -e
         echo 'Waiting for Postgres to be available'
-        export PGPASSWORD="$$POSTGRES_ENV_POSTGRES_PASSWORD"
+        export PGPASSWORD=$${POSTGRES_ENV_POSTGRES_PASSWORD}
         maxTries=10
-        while [ "$$maxTries" -gt 0 ] && ! psql -h "$$DB_HOST" -U 'postgres' -c '\l'; do
+        while [[ maxTries -gt 0 ]] && ! psql -h $${DB_HOST} -U 'postgres' -c '\l'; do
```
Comment (passga) [inline modules/runtime/pages/bonita-docker-installation.adoc]: in 2021.2, the psql tool is no more installed in docker image so we could not use it to wait for that database to be available.

File: modules/runtime/pages/bonita-docker-installation.adoc [hunk 2]
```diff
@@ -128,7 +128,7 @@ docker run --name bonita_postgres -h <hostname> -v bonita-lic:/opt/bonita_lic/ -
 
 ==== Using docker-compose
 
-Create a file stack.yml with the following content (remember to replace the `<hostname>` with the one used in the licence generation command):
+Create a file `docker-compose.yml` with the following content
 
 [source,shell script,subs="+macros"]
```
Comment (alachambre) [inline modules/runtime/pages/bonita-docker-installation.adoc]: It seems that the source code below is YML and not shell script. 
You should update it, else the comment on the first line doesn't looks like a comment.

File: modules/runtime/pages/bonita-docker-installation.adoc (hunk 1 above)
Comment (baptistemesta) [reply modules/runtime/pages/bonita-docker-installation.adoc]: Yes, It's not available anymore in 2022.1, I planned to change it in that version

File: modules/runtime/pages/bonita-docker-installation.adoc (hunk 2 above)
Comment (baptistemesta) [reply modules/runtime/pages/bonita-docker-installation.adoc]: Nice catch! thanks

File: modules/runtime/pages/bonita-docker-installation.adoc (hunk 1 above)
Comment (passga) [reply modules/runtime/pages/bonita-docker-installation.adoc]: sorry  you right, i did mistake with version number :(

File: modules/runtime/pages/bonita-docker-installation.adoc [hunk 3]
```diff
@@ -168,25 +168,23 @@ services:
...
+        export PGPASSWORD=$${POSTGRES_ENV_POSTGRES_PASSWORD}
         maxTries=10
-        while [ "$$maxTries" -gt 0 ] && ! psql -h "$$DB_HOST" -U 'postgres' -c '\l'; do
+        while [[ maxTries -gt 0 ]] && ! psql -h $${DB_HOST} -U 'postgres' -c '\l'; do
             let maxTries--
             sleep 1
         done
-        echo
-        if [ "$$maxTries" -le 0 ]; then
+        if [[ maxTries -le 0 ]]; then
```
Comment (passga) [inline modules/runtime/pages/bonita-docker-installation.adoc]: maxTries should be define as variable `$maxTries`, no ?

File: modules/runtime/pages/bonita-docker-installation.adoc (hunk 1 above)
Comment (passga) [reply modules/runtime/pages/bonita-docker-installation.adoc]: maxTries should be define as variable $maxTries, no ?

File: modules/runtime/pages/bonita-docker-installation.adoc (hunk 3 above)
Comment (baptistemesta) [reply modules/runtime/pages/bonita-docker-installation.adoc]: it's define above (172)

Comment (baptistemesta) [reply modules/runtime/pages/bonita-docker-installation.adoc]: I reverted that part: both `"$$maxTries"` and `maxTries` work. Not `$maxTries`

Comment (passga) [reply modules/runtime/pages/bonita-docker-installation.adoc]: ok

**Review by passga — APPROVED** [review]
LGTM!!

**Pull request merged**